settings/Developer.setting.php

```diff
@@ -48,10 +48,10 @@
     'quick_form_type' => 'YesNo',
     'default' => '0',
     'add' => '4.3',
-    'title' => ts('Enable Drupal Watchdog Logging'),
+    'title' => ts('Send CiviCRM errors to CMS logs?'),
     'is_domain' => 1,
     'is_contact' => 0,
-    'description' => ts('Set this value to Yes if you want CiviCRM error/debugging messages to appear in the Drupal error logs.'),
+    'description' => ts('Set this value to Yes if you want CiviCRM error/debugging messages to appear in your CMS error logs.'),
     'help_text' => "Set this value to Yes if you want CiviCRM error/debugging messages the appear in your CMS' error log. In the case of Drupal, this will cause all CiviCRM error messages to appear in the watchdog (assuming you have Drupal's watchdog enabled)",
```

Comment:
This says "the appear" instead of "to appear". I am aware that you didn't change this and it may be outside of the scope of your PR, just something I noticed while looking at this.

Reply:
Good spot!